src/utils/omit-tags.test.ts

```diff
@@ -0,0 +1,6 @@
+import { omitTags } from "./omit-tags";
+
+test("omitPTags returns summary without <p> and </p> tags", () => {
```

Comment:
Is the test comment accurate to the functionality of omitTags? Does it not also work for <b> and </b> and others? The test comment should be reflective of that